docs/reference-guides/block-api/block-styles.md

```diff
@@ -0,0 +1,118 @@
+# Styles
+
+Block Styles allow alternative styles to be applied toexisting blocks. They work by adding a className to the block's wrapper. This className can be used to provide an alternative styling for the block if the block style is selected. See the [Getting Started with JavaScript tutorial](/docs/how-to-guides/javascript/) for a full example.
+
+_Example:_
+
+```js
+wp.blocks.registerBlockStyle( 'core/quote', {
+	name: 'fancy-quote',
+	label: 'Fancy Quote',
+} );
+```
+
+The example above registers a block style named `fancy-quote` to the `core/quote` block. When the user selects this block style from the styles selector, an `is-style-fancy-quote` className will be added to the block's wrapper.
+
+By adding `isDefault: true` you can mark the registered block style as the one that is recognized as active when no custom class name is provided. It also means that there will be no custom class name added to the HTML output for the style that is marked as default.
```

Comment:
_no custom class name added to the HTML output for the style that is marked as default._
Please help confirm that this is correct. When there are custom styles available, and I select the default style,
I am seeing this CSS class both in the editor and front `is-style-default`

Reply:
Good catch, I'll do some digging to confirm that this is accurate.